.gitlab-ci.yml

```diff
@@ -180,6 +180,16 @@ test:debug:
     reports:
       junit: "**/build/test-results/testDebugUnitTest/*.xml"
 
+test:debug:
+  <<: *test-debug-template
+
+test:debug:itr:
```

Comment:
isn't this going to increase considerably the execution time in the CI ?

Reply:
This will increase compute time but not the wall time, as the new jobs will be running in parallel with the existing ones.
The idea is to have it like this for some time to ensure that the ITR-enabled jobs work adequately, and only after that to remove the old jobs.